### PR TITLE
Minor change in Sensors

### PR DESCRIPTION
**Pull request by umarmanzoor**
Minor change in Sensors

### CODE REVIEW

File: saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/mSpRL2017App.scala [hunk 1]
```diff
@@ -73,4 +73,7 @@ object SpatialOntologyApp extends App {
   val lmCandidateRelations = getCandidateRelations[Token](lmCandidates, spCandidates)
   relations.populate(trCandidateRelations ++ lmCandidateRelations)
 
+
+
+
```
Comment (kordjamshidi) [inline saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/mSpRL2017App.scala]: a lot of empty lines are added here.

Comment (umarmanzoor) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/mSpRL2017App.scala]: @Rahgooy empty lines.... its your code...

Comment (Rahgooy) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/mSpRL2017App.scala]: @umarmanzoor it's not mine. The last version in my repo is this: https://github.com/Rahgooy/saul/blob/master/saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/mSpRL2017App.scala

Comment (umarmanzoor) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/mSpRL2017App.scala]: @Rahgooy ok, so should I remove the empty lines, commit and push?

**Pull request merged**